app/views/reports/satisfaction_summary.html.erb

```diff
@@ -0,0 +1,65 @@
+<div class="row">
+  <div class="col-md-12">
+    <h1>Satisfaction</h1>
+  </div>
+</div>
+<div class="row">
+  <div class="col-md-12">
+    <div class="well">
+      <%= form_for @satisfactions, url: reports_satisfaction_summary_path, method: :get, html: { class: 'inline form-inline' } do |f| %>
+        <div class="form-group">
+          <%= f.label :start_date, 'From', class: 'inline' %>
+          <%= f.text_field :start_date, type: :date, class: 'form-control t-start', placeholder: 'dd/mm/yyyy' %>
+        </div>
+        <div class="form-group">
+          <%= f.label :end_date, 'To', class: 'inline' %>
+          <%= f.text_field :end_date, type: :date, class: 'form-control t-end', placeholder: 'dd/mm/yyyy' %>
+        </div>
+        <%= f.submit 'Search', class: 'btn btn-default t-search' %>
+      <% end %>
+
+      <%= form_for @satisfactions, url: reports_satisfaction_summary_path(format: :csv), method: :get, html: { class: 'inline form-inline' } do |f| %>
+        <%= f.hidden_field :start_date %>
+        <%= f.hidden_field :end_date %>
+        <%= f.submit 'Export CSV', class: 'btn btn-default t-export-csv' %>
+      <% end %>
+
+      <%= form_for @satisfactions, url: reports_satisfaction_path(format: :csv), method: :get, html: { class: 'inline form-inline' } do |f| %>
+        <%= f.hidden_field :start_date %>
+        <%= f.hidden_field :end_date %>
+        <%= f.submit 'Export raw CSV', class: 'btn btn-default t-export-raw-csv' %>
+      <% end %>
+    </div>
+  </div>
+</div>
+</div>
+<div class="row">
+  <div class="col-md-12">
+    <table class="table table-striped table-bordered">
+      <thead>
+        <tr class="table-header">
+          <th scope="col"></th>
+          <th>CAS</th>
+          <th>CITA</th>
+          <th>NICAB</th>
+          <th>TPAS</th>
+          <th>Total</th>
+          <th>Weighted average</th>
+        </tr>
+      </thead>
+      <tbody>
+        <%- @satisfaction_summary.rows.each do |row| %>
+          <tr class="t-row">
+            <td><%= row.name %></td>
+            <td class="text-right"><%= row.cas %></td>
+            <td class="text-right"><%= row.cita %></td>
+            <td class="text-right"><%= row.nicab %></td>
+            <td class="text-right"><%= row.tp %></td>
```

Comment:
Are we still planning to do the `tp` => `tpas` change separately? We're adding quite a lot of code that references it like this now.